src/script/script.h

```diff
@@ -600,6 +599,7 @@ CScript BuildScript(Ts&&... inputs)
             // Otherwise invoke CScript::operator<<.
             ret << input;
         }
+        cnt++;
```

Comment:
```suggestion
        ++cnt;
```

Reply:
@darosior did you want to address this comment?

Reply:
I was not planning to unless anyone felt strongly otherwise, as it's really a nit..

Reply:
If you're interested (for now or later), in addition to https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c there is further rationale in https://lefticus.gitbooks.io/cpp-best-practices/content/08-Considering_Performance.html#prefer-i-to-i.

Reply:
I haven't yet seen an iterator type where this isn't optimized to the same assembly. Though, if we want this, it might be better enforced with a tidy check than with review comments that invalidate review

Reply:
If we don't want this, it should be removed from the developer notes.

Reply:
Agree. See also https://github.com/bitcoin/bitcoin/pull/24846#issuecomment-1098774606

Reply:
Choosing `++v;` over `v++;` is generally good advice if you don't want to go into details, but in practically it really matters for performance in case `v` is a non-trivial, big, type. In this case where it's just an integer it won't make to any difference in the resulting binary code with any compiler created the past 20 years.